release-please-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "bootstrap-sha": "62a4ce414f5b02f539902f88dd4b548df5cfa8ef",
```

Comment:
Does this bootstrap SHA need to be diffent @ascorbic? I copied this config from the next-runtime project.

Reply:
Looks like I need to generate this with the release please.

Reply:
https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#starting-commit

Looks like this isn't required, and is only added when you want to start the release please process from a particular git SHA. I'd use the one that main is at currently.